sdk/src/system/DateTime.cpp

```diff
@@ -452,4 +451,4 @@ std::string DateTime::toString(const std::string& in_format) const
 
 } // namespace system
 } // namespace launcher_plugins
-} // namespace rstudio
+} // namespace rstudio
```

Comment:
Nit: blank last line
```suggestion
} // namespace rstudio

```